packages/components/package.json

```diff
@@ -44,6 +45,7 @@
     "@storybook/react": "^6.3.12",
     "@types/classnames": "^2.2.11",
     "@types/jest": "^26.0.15",
+    "@types/mocha": "^9.0.0",
```

Comment:
I think this was added by mistake. We aren't using Mocha AFAIK. Can you remove this?

Reply:
Yes, this happened while I trying to setup the test environment, I will remove this 👍